dmg2john Warning from CircleCI Wine64

I've never seen this before.
```
dmg2john.c: In function 'hash_plugin_parse_hash':
dmg2john.c:252:17: warning: overflow in conversion from 'long long unsigned int' to 'long int' changes value from '18446744073709550340' to '-1276' [-Woverflow]
   if (lseek(fd, -sizeof(cencrypted_v1_header), SEEK_END) < 0) {
                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/src/dmg2john.c
+++ b/src/dmg2john.c
@@ -249,7 +249,7 @@ static void hash_plugin_parse_hash(char *in_filepath)
 
 	// fprintf(stderr, "Header version %d detected\n", headerver);
 	if (headerver == 1) {
-		if (lseek(fd, 0 - (off_t)sizeof(cencrypted_v1_header), SEEK_END) < 0) {
+		if (lseek(fd, -(off_t)sizeof(cencrypted_v1_header), SEEK_END) < 0) {
 			fprintf(stderr, "Unable to seek in %s\n", filename);
 			goto bailout;
 		}

--- a/src/dmg2john.c
+++ b/src/dmg2john.c
@@ -249,7 +249,7 @@ static void hash_plugin_parse_hash(char *in_filepath)
 
 	// fprintf(stderr, "Header version %d detected\n", headerver);
 	if (headerver == 1) {
-		if (lseek(fd, -sizeof(cencrypted_v1_header), SEEK_END) < 0) {
+		if (lseek(fd, 0 - (off_t)sizeof(cencrypted_v1_header), SEEK_END) < 0) {
 			fprintf(stderr, "Unable to seek in %s\n", filename);
 			goto bailout;
 		}

